frontend/admin/src/js/components/pool/UpdatePool.tsx

```diff
@@ -152,6 +152,16 @@ export const UpdatePoolForm: React.FunctionComponent<UpdatePoolFormProps> = ({
     }),
   );
 
+  const operationalRequirementsSubsetV2 = [
+    OperationalRequirement.OvertimeOccasional,
+    OperationalRequirement.OvertimeRegular,
+    OperationalRequirement.ShiftWork,
+    OperationalRequirement.OnCall,
+    OperationalRequirement.Travel,
+    OperationalRequirement.TransportEquipment,
+    OperationalRequirement.DriversLicense,
+  ];
```

Comment:
Pretty minor, but still seeing OVERTIME_SCHEDULED show up in the stories, which is a bit of a pain, since the faker stuff doesn't take into account the variations :/

<img width="1137" alt="Screen Shot 2022-07-13 at 1 11 28 PM" src="https://user-images.githubusercontent.com/305339/178791925-e51d1737-e6a4-4dd2-9831-eedb8b475465.png">

https://61e099a1bb1465003a73d3ce-qlakyjhnjl.chromatic.com/?path=/story/pool-candidates--update-pool-candidate-form

Reply:
I wouldn't worry too much about faker not accounting for different versions, at least I think it isn't something to worry about

Reply:
oh hey, it's hardcoded in stories, so easy to fix :)

Reply:
[got it](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3293/commits/bcd8023bdd04ea30a139d8b55c2386f37105878f)